application/forms/Config/General/ApplicationConfigForm.php

```diff
@@ -70,21 +70,32 @@ public function createElements(array $formData)
             )
         );
 
-        $this->addElement(
-            'select',
-            'global_config_backend',
-            array(
-                'required'      => true,
-                'autosubmit'    => true,
-                'label'         => $this->translate('User Preference Storage Type'),
-                'multiOptions'  => array(
-                    'ini'   => $this->translate('File System (INI Files)'),
-                    'db'    => $this->translate('Database'),
-                    'none'  => $this->translate('Don\'t Store Preferences')
+        if ($this->getRequest()->getModuleName() != 'setup'
+            && $formData['global_config_backend'] === 'ini'
```

Comment:
this form should not be displayed in the setup and should only be displayed in the configuration -> application if the ini file is recognized

Reply:
Okay, fine. But the wizard doesn't let you choose the type anymore at all, so it can't be of type INI and therefore the second condition should suffice, shouldn't it?

Reply:
yes, you are right, but we have to add a check if that variable exists.